File: python/0521.最长特殊序列Ⅰ.py
class Solution:
    # 1. O(1) t:28ms(88%) O(1) m:15M(45%) 脑筋急转弯
    def findLUSlength(self, a: str, b: str) -> int:
        return max(len(a), len(b)) if a != b else -1

File: cpp/0521.最长特殊序列Ⅰ.cpp
class Solution {
public:
    // 1. O(1) t:0ms(100%) O(1) m:6M(58%) 脑筋急转弯
    int findLUSlength(string a, string b) {
        return a != b ? max(a.length(), b.length()) : -1;
    }
};